biodatasets/codiesp/codiesp.py

```diff
@@ -0,0 +1,310 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+A dataset loading script for the CODIESP corpus.
+
+The CODIESP dataset is a collection of 1,000 manually selected clinical 
+case studies in Spanish that was designed for the Clinical Case Coding 
+in Spanish Shared Task, as part of the CLEF 2020 conference. This community
+task was divided into 3 sub-tasks: diagnosis coding (CodiEsp-D), procedure
+coding (CodiEsp-P) and Explainable AI (CodiEsp-X).
+"""
+
+import os
+from collections import defaultdict
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from utils import schemas
+from utils.configs import BigBioConfig
+from utils.constants import Tasks
+
+_CITATION = """\
+@inproceedings{
+  title={Overview of automatic clinical coding: annotations, guidelines, and \
+  solutions for non-english clinical cases at codiesp track of CLEF eHealth 2020},
+  author={Miranda-Escalada, Antonio and Gonzalez-Agirre, Aitor and Armengol-Estap{\'e}, Jordi and Krallinger, Martin},
+  booktitle={Working Notes of Conference and Labs of the Evaluation (CLEF) Forum. CEUR Workshop Proceedings},
+  year={2020}
+}
+"""
+
+_DATASETNAME = "cantemist"
```

Comment:
Why `cantemist` and not `codiesp`?

Reply:
ups, I mixed things (cantemist is another dataset I was working on)